modules/apps/portal/portal-monitoring/src/main/java/com/liferay/portal/monitoring/internal/processor/PortalRequestDataSampleProcessor.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.monitoring.internal.processor;
+
```

Comment:
HEy @RenanUmV, please rename the package to `com.liferay.portal.monitoring.internal.data.sample.processor` instead. Thx